wasm/jsapi/table/constructor-reftypes.tentative.any.js

```diff
@@ -0,0 +1,49 @@
+// META: global=window,dedicatedworker,jsshell
+// META: script=assertions.js
+// META: script=/wasm/jsapi/wasm-constants.js
```

Comment:
Was this intentional? I'm not seeing a `wasm-constants.js` file in this location?

Reply:
Oh, this was removed in #16326. Will fix.